hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/KafkaSource.java

```diff
@@ -50,6 +56,10 @@ protected KafkaSource(TypedProperties props, JavaSparkContext sparkContext, Spar
     this.metrics = metrics;
   }
 
+  protected boolean shouldAppendKafkaOffsets() {
+    return KafkaOffsetPostProcessor.class.getName().equals(props.getString(SCHEMA_POST_PROCESSOR_PROP, null));
```

Comment:
there could be multiple entries in the schema post processor config right ? should we not parse the config value and find if "KafkaOffsetPostProcessor.class.getName()" is one of them ?

Reply:
I have no idea, but I updated it to do what you suggest